xla/service/gpu/backend_configs.proto

```diff
@@ -61,6 +61,10 @@ message CudnnConvBackendConfig {
   // Serialization of the graph described by the convolution and adjacent
   // pointwise ops.
   optional string serialized_graph = 9;
+
+  //Call context which specifies whether convolution operation 
```

Comment:
nit: Please leave a space after the "//" and the first word (same in the line below)